src/test/resources/testscripts/NextflowRnaseq.sh

```diff
@@ -53,12 +53,18 @@ terra nextflow -version
 #      google.location = 'europe-west2'
 #      google.project = "$GOOGLE_CLOUD_PROJECT"
 #      google.region  = 'europe-west1'
+#      google.lifeSciences.serviceAccountEmail = "$GOOGLE_SERVICE_ACCOUNT_EMAIL"
+#      google.lifeSciences.network = 'network'
+#      google.lifeSciences.subnetwork = 'subnetwork'
```

Comment:
probably worth a comment that these are special values based on the networks created by RBS today for workspace projects.